posthog-react-native/src/posthog-rn.ts

```diff
@@ -126,10 +86,14 @@ export class PostHog extends PostHogCore {
           await this.captureNativeAppLifecycleEvents()
         }
       }
+
       await this.persistAppVersion()
-    }
+    })
+  }
 
-    this._setupPromise = setupAsync()
+  // NOTE: This is purely a helper method for testing purposes or those who wish to be certain the SDK is fully initialised
+  public async ready(): Promise<void> {
+    await this._initPromise
```

Comment:
We could check `_isInitialized` first here.

Reply:
Relates to https://github.com/PostHog/posthog-js-lite/pull/189/files#r1502255807